packages/python-aiosmtplib/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-aiosmtplib
+_pkgname=aiosmtplib
+pkgver=2.0.0
+pkgrel=1
+pkgdesc='Asynchronous SMTP client for use with asyncio.'
+arch=('any')
+url='https://github.com/cole/aiosmtplib'
+license=('MIT')
+depends=('python')
+makedepends=('python-setuptools')
```

Comment:
do not use deprecated setup-tools when possible, upstream has a https://github.com/cole/aiosmtplib/blob/main/pyproject.toml, use https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-PEP517 instead

Reply:
PKGBUILD-python-lib-PEP517 template applied.